### PR TITLE
add some basic ideas for 2015

### PR DESCRIPTION
**Pull request by Kennric**

### CODE REVIEW

File: source/gsoc/ideas2015.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Basic understanding of the Django application framework, ability to program in
+Python, and familiarity with mobile applications, PhoneGap or related tools, and 
+REST APIs.
+
+**Mentor:** Ken (kennric)
+
+------
+
+.. _gsoc2014-other-projects:
```
Comment (ramereth) [inline source/gsoc/ideas2015.rst]: You should change this to 2015 :)

File: source/gsoc/ideas2015.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+.. _gsoc2015-whats-fresh:
+
+What's Fresh
+~~~~~~~~~~~~
+
+Project: What's Fresh version 2.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What's Fresh is an open source project originally funded by the Oregon Sea Grant
```
Comment (ramereth) [inline source/gsoc/ideas2015.rst]: You should add a link to the github repo and maybe a link to Oregon Sea Grant's site.

File: source/gsoc/ideas2015.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+to create a mobile app for locating fresh local seafood on the Oregon coast. 
+Version 1.0 was created by a collaboration between OSU's Business Solutions Group
```
Comment (ramereth) [inline source/gsoc/ideas2015.rst]: Add a link to the BSG's site.

File: source/gsoc/ideas2015.rst [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+and the OSL under the umbrella of OSU's Center for Applied Systems and Software. It
```
Comment (ramereth) [inline source/gsoc/ideas2015.rst]: Add the acronym CASS and link to the website.

**Comment by ramereth** [comment]
@Kennric looks good otherwise. Make sure you update the bottom of the gsoc index page to list these projects.

**Comment by Kennric** [comment]
Added some links and a sentence about bioinformatics to the PGD description. I am goign to go ahead and merge so we can get our application material in.

**Pull request merged**